package.json

```diff
@@ -6,7 +6,8 @@
     "packages/*"
   ],
   "scripts": {
-    "start": "openmrs develop --sources 'packages/esm-*-app'",
+    "start": "openmrs develop --backend https://ugandaemr-backend.mets.or.ug --sources 'packages/esm-*-app'",
+    "start2": "openmrs develop --backend https://ohri-dev.globalhealthapp.net --sources 'packages/esm-*-app'",
```

Comment:
@pirupius why do we need start2

Reply:
Same comment as above. `ugandaemr` backend hadn't been updated.